apps/web/src/services/asset-inventory/collector/collector-create/modules/MultipleProviderForm.vue

```diff
@@ -56,6 +58,7 @@ import {
 const collectorFormStore = useCollectorFormStore();
 const collectorFormState = collectorFormStore.$state;
 
+const emit = defineEmits<{(event: 'update:is-provider-valid', value: boolean): void; }>();
```

Comment:
Suggestion: How about `update-valid` like something not having `update:-` prefix?
I think, using `update:-` appears to have some bound state.

Reply:
Oh. I love it.